generated/advisor/microsoft.advisor/2016-07-12-preview/types.md

```diff
@@ -1,5 +1,13 @@
 # Microsoft.Advisor @ 2016-07-12-preview
 
+## Resource Microsoft.Advisor/generateRecommendations@2016-07-12-preview (ReadOnly)
```

Comment:
Strange one! It's clearly not useful in a practical sense, but it's following the resource GET URI format, so I suppose we should support it (and don't really have any decent heuristics to exclude it).

Reply:
This request wasn't marked as a resource by the RP, so per the update discussed in https://github.com/Azure/bicep-types-az/pull/838#pullrequestreview-986978807, it was removed from the changeset.